sdk/node/doc/assets/css/main.css

```diff
@@ -1,871 +0,0 @@
-/*! normalize.css v1.1.3 | MIT License | git.io/normalize */
-/* ========================================================================== HTML5 display definitions ========================================================================== */
```

Comment:
What I mean is my patch accidentally picked up some of those problems related to files being added/removed.  The only thing I meant to change under sdk/node was the addition of sdk/node/Makefile.  I'll push an update based on the latest HEAD.